### PR TITLE
Add bing translator converter

### PR DESCRIPTION
**Pull request by bobeaton**
this isn't quite ready -- the help setup file needs to be fixed and the UI for the Setup tab needs to be tested. But the new "BingTranslator" EncConverter itself is working. (just switching computers)

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -56,7 +56,7 @@ jobs:
         sudo apt-get update
 
     - name: Install dependencies
-      run: sudo apt-get install --assume-yes automake g++ python-dev libicu-dev mono5-sil icu-dev-fw libteckit-dev
+      run: sudo apt-get install --assume-yes automake g++ python-dev libicu-dev mono5-sil icu-dev-fw libteckit-dev Newtonsoft.Json
```
Comment (jasonleenaylor) [inline .github/workflows/build.yml:59]: This shouldn't be needed if you are adding through Nuget

File: src/EcTranslators/EcTranslators.csproj [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+  <ItemGroup>
+    <None Include="app.config" />
+    <None Include="FieldWorks.snk" />
+    <None Include="Properties\Settings.settings">
+      <Generator>SettingsSingleFileGenerator</Generator>
+      <LastGenOutput>Settings.Designer.cs</LastGenOutput>
+    </None>
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json">
```
Comment (jasonleenaylor) [inline src/EcTranslators/EcTranslators.csproj:128]: This kind of reference is meant for new style .csproj files, but I suppose it works this way for windows builds using VS2019.
If you added Newtonsoft.Json to the EncCnvtrs\packages.config file and referenced the assembly with a normal package reference it should fix your linux build.

File: src/EncCnvtrs/packages.config [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
-	<package id="Microsoft.Web.WebView2" version="1.0.992.28" targetFramework="net48" />
-	<package id="NUnit" version="3.13.2" targetFramework="net472" />
-	<package id="NUnit3TestAdapter" version="4.0.0" targetFramework="net472" />
+  <package id="Geckofx60.32" version="60.0.55" targetFramework="net48" />
+  <package id="Geckofx60.32.Linux" version="60.0.49" targetFramework="net471" />
+  <package id="Geckofx60.64" version="60.0.55" targetFramework="net48" />
+  <package id="Geckofx60.64.Linux" version="60.0.51" targetFramework="net471" />
+  <package id="Microsoft.Web.WebView2" version="1.0.992.28" targetFramework="net48" />
+  <package id="NUnit" version="3.13.2" targetFramework="net472" />
+  <package id="NUnit3TestAdapter" version="4.0.0" targetFramework="net472" />
```
Comment (jasonleenaylor) [inline src/EncCnvtrs/packages.config:9]: I don't see Newtonsoft.Json here.

File: src/EcTranslators/EcTranslators.csproj (hunk 2 above)
Comment (bobeaton) [reply src/EcTranslators/EcTranslators.csproj:128]: my earlier attempts (if you look higher in the pull request commit list) had it being pulled by nuget in the packages.config file, first v13 and when that didn't work, v12, then adding a redirect for it to v12 in the app.config... none of those permutations worked. So all the latter ones you discuss here are "reaching at straws" trying to make it work. I can revert it back to that scenario (in packages.config) and someone can try to fix it from there?

**Pull request merged**